test/puppeteer/demo.test.ts

```diff
@@ -96,7 +113,7 @@ it('reroutes enter presses on menu items as clicks', async () => {
 	await menuOpen();
 
 	// eslint-disable-next-line @typescript-eslint/no-misused-promises
-	page.on('dialog', async dialog => {
+	page.once('dialog', async dialog => {
```

Comment:
Why change from `on` to `once` here?

Reply:
Listeners were persisting across tests, so they either needed to be specifically removed at the end of each test, but I thought this was a cleaner solution, since we only need the listener for one test.